### ISSUE TITLE
takers academy ... £4,000 for the Remote Course!!

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/nelsonic.github.io#263]
http://www.makersacademy.com/ is charging £4,000 for their _remote_ course:
![takers-academy-pricing](https://cloud.githubusercontent.com/assets/194400/21489309/ed894d4c-cbe0-11e6-85a9-9dca4b05a1d3.png)

Does anyone `else` see a _problem_ with this...?

> More than ***half world's people*** (_still_) don't have _access_ to the internet. http://www.internetworldstats.com/stats.htm and 
it's estimated that ***3 Billion People*** (_still_) live on _less_ than **$2.50 per day**. https://openknowledge.worldbank.org/bitstream/handle/10986/25078/9781464809583.pdf
_Granted_ at that income level people aren't going to have cash for a _computer_ in a hurry, so they are _unlikely_ to be writing web applications ... but if they were _able_ to save together the cash to afford a **Raspberry Pi** ([$35](https://www.raspberrypi.org/blog/raspberry-pi-3-on-sale/)) they _could_ start learning ... and if their village/town had an [internet cafe](http://www.businessdailyafrica.com/Solar-powered-Internet-cafes-link-rural-Africa/-/539444/1195700/-/8iugjb/-/index.html) they would be able to _join_ the https://en.wikipedia.org/wiki/Digital_Revolution
None of this is _possible_ if learning materials are hidden behind a "_Pay Wall_"... 😞 

# What is the `Marginal Cost` of running a Remote Training Program? Could they sell it for say £200 instead...? How about `Free`...?

**Comment by nelsonic** [comment]
# Makers' <> Phoenix in 2018 ?

What is the "bet" that Makers' "transition" to Phoenix by Summer of 2018 ?

Their **curriculum** http://www.makersacademy.com/curriculum/ `while` well-defined appears to be **_stuck_ in 2009**:
![image](https://cloud.githubusercontent.com/assets/194400/22854613/572d7824-f06a-11e6-89e3-876ea548531c.png) <br />
As if the internet didn't "_move on_" from Ruby-on-Rails and JQuery ... 🤔 

> I wager that Makers' will "_see the light_" at _some point_ in **Mid 2017** and start having "_discussions_" about it towards the Autumn (_always in private of course because **why** would you want to be **transparent** about anything...?_) and `.then` ***maybe*** by 2018 they will be including Phoenix in their curriculum.

On a side note, have a look at their GitHub: https://github.com/makersacademy
![makers github](https://cloud.githubusercontent.com/assets/194400/22854584/b00e2818-f069-11e6-8882-fac4ad84d66a.png)

***None*** of their "_core_" learning material is public. 
only a handful of basic "_challenges_" are public so that their students can fork/clone them.
and only 34 people have made their membership of the Org public.
What does that _tell_ you about their ***culture***...?

By _contrast_ ***everything*** we do is **100% Open Source** and for the benefit of the _community_ not the "_investors_": https://github.com/dwyl snapshot (_incognito mode_)
![image](https://cloud.githubusercontent.com/assets/194400/22854605/2dab8284-f06a-11e6-80d3-e7381915449c.png)

Meanwhile we are going to all be "_Boss Level_" in ["***New Stack***"](https://github.com/dwyl/technology-stack/issues/37) by **April 2017** and have 3 ***Complete*** Open Source Projects available on GitHub! 
So... _actually_, maybe Makers' _won't_ adopt Phoenix, because they will _probably_ see it as ["NIH"](https://github.com/dwyl/technology-stack/issues/38) and "_stick with what they know and are good at ..._" _i.e. Rails & JQuery..._

# Why Does This _Matter_?

It doesn't it's just a little prediction. 
I'm not going to change what I'm doing because _anyone_ `else` is trying something different or choosing to stick with what they know.
Just going to stay focussed on building and watch what happens in 2017/18 😉

**Comment by nelsonic** [comment]
http://constructorlabs.com/ **£416.67 per _week_** plus living expenses ...
![image](https://user-images.githubusercontent.com/194400/37090145-1336783c-21fb-11e8-9cc8-b28d56011091.png)

http://constructorlabs.com/course
![image](https://user-images.githubusercontent.com/194400/37090184-3b5c56ec-21fb-11e8-90b6-e268a2c9f340.png)

via: https://twitter.com/iamdevloper/status/970967017401577472
![image](https://user-images.githubusercontent.com/194400/37090202-500b1434-21fb-11e8-92b8-f92f75718a73.png)

**Comment by nelsonic** [comment]
https://vanillajsacademy.com/#ready-to-buy
![image](https://user-images.githubusercontent.com/194400/45182848-559c4a80-b21a-11e8-816c-b52a43d0db93.png)
![image](https://user-images.githubusercontent.com/194400/45182866-5e8d1c00-b21a-11e8-9cc3-f7df5260da72.png)

**Comment by nelsonic** [comment]
https://lambdaschool.com
![lambdaschool-20k](https://user-images.githubusercontent.com/194400/45252054-c80b4880-b348-11e8-9317-1325d0109474.png)

It's "_free_" and we "own you" (17% of income!) for two years or **$20k up-front**.

I can see how this would be "_appealing_" to people. 
If I had "no money" and wanted a _structured_ way to learn web skills
with a reasonably guaranteed "job opportunity" at the end, 
this would appeal.

# Syllabus?

Want to know what we cover?
Give us your personal data first!

![image](https://user-images.githubusercontent.com/194400/45252200-b165f100-b34a-11e8-80d1-0ae7fa6c633a.png)

> https://learn.lambdaschool.com/syllabus/cs-fsw

I still thinking learning/education should be free as in "sun light"
not "_free until you pay us back_" ...

**Comment by nelsonic** [comment]
https://careerfoundry.com/en/courses/become-a-web-developer 
**€3,324** up-front or **€3,499** ("payment plan")
![image](https://user-images.githubusercontent.com/194400/45255059-c5751700-b379-11e8-9842-0447b4867299.png)

@iteles Apparently you follow them on Twitter:
![image](https://user-images.githubusercontent.com/194400/45255085-0bca7600-b37a-11e8-91f5-44d5b09fffbd.png)

**Comment by iteles** [comment]
Yes, I follow a few of these types of 'bootcamps' but I haven't seen a post from them in my feed possibly ever so engagement probably isn't high (at least from me).

**Comment by nelsonic** [comment]
Ok, these guys win the prize for the most over-priced: https://www.oxford-royale.co.uk/apply
![image](https://user-images.githubusercontent.com/194400/54319862-f6612d80-45e2-11e9-85cf-967378c58811.png)

£5k for **_two_ weeks** of "summer school" ... 😮 💸
